include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -332,7 +332,7 @@ class CommonOperatorLoader : public ProtobufLoader {
     unsigned_t k = loadInt(dict["k"]);
 
     int axis = dict.count("axis") ? loadInt(dict["axis"]) : -1;
-    unsigned_t lastDim = in.dims().size() - 1;
+    int lastDim = (unsigned_t)in.dims().size() - 1;
```

Comment:
I'm not sure why I changed this to `unsigned_t` 😃 -- it should be fine just as an `int`, without the cast, or even `ssize_t`.

Reply:
Done.